src/components/pages/MissionControl/Checkbox.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import unchecked_box from '../../../assets/icons/unchecked_box.svg';
+import checked_box from '../../../assets/icons/checked_box.svg';
+
+const Checkbox = props => {
+  console.log(props);
```

Comment:
I know they don't want us to have comments in the code but do they want us to take out the console.logs too?

Reply:
Yeah I think so, you don't really want any client-facing console logs

Reply:
Thanks, I totally forgot to remove those